test/test_other.py

```diff
@@ -9561,6 +9561,18 @@ def test_wasm_sourcemap_dead(self):
     # has only two entries
     self.assertRegexpMatches(output, r'"mappings":\s*"[A-Za-z0-9+/]+,[A-Za-z0-9+/]+"')
 
+  def test_wasm_sourcemap_resolve_dummy_comp_dir(self):
+    wasm_map_cmd = [PYTHON, path_from_root('tools/wasm-sourcemap.py'),
+                    '--dwarfdump-output',
+                    test_file('other/wasm_sourcemap_resolve_dummy_comp_dir/foo.wasm.dump'),
+                    '-o', 'a.out.wasm.map',
+                    test_file('other/wasm_sourcemap_resolve_dummy_comp_dir/foo.wasm'),
+                    '--basepath=' + path_from_root()]
+    self.run_process(wasm_map_cmd, stdout=PIPE, stderr=PIPE)
```

Comment:
Maybe don't capture stdout and stderr here since if this command ever fails it could be good to show the output.